components/edit_post_modal/edit_post_modal.tsx

```diff
@@ -448,7 +448,6 @@ export class EditPostModal extends React.PureComponent<Props, State> {
     handleEntered = () => {
         if (this.editbox) {
             this.editbox.focus();
-            this.editbox.recalculateSize();
```

Comment:
I had to remove this since the `EditPostModal` no longer has access to the `AutosizeTextarea`'s methods, but that's fine since this worked correctly without it (ie. it sized the modal and scrolled the textarea to the end on long posts)